_pages/about-peer-review.md

```diff
@@ -1,46 +1,98 @@
+    excerpt: "More here"
+    title: "Scientists need vetted, maintained open source tools"
+benefits_scientists:
+  - image_path: images/pyopensci-people-at-computer.jpg
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "Scientists need trusted and vetted tools to support their open science workflows. Through our [partnerships with domain specific communities](/partners.html) our catalog of trusted tools for scientists will grow. "
+    title: "Benefits of software peer review for scientists"
+benefits_maintainers:
+  - image_path: images/pyopensci-people-at-computer.jpg
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "Peer review provides a form of academic credit currently unavailable to scientists developing software. Further, peer review multiplies shared knowledge, such as navigating the Python packaging ecosystem, which is challenging, even for seasoned Pythonistas. Our diverse community supports scientific package maintainers in their efforts to develop and build robust software."
+    title: "Benefits for open source maintainers"
+peer_review:
+  - image_path: images/pyopensci-people-at-computer.jpg
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "Software peer review, similar to review of scientific papers, is a process where scientists vet software code, documentation and infrastructure. pyOpenSci leads an [open peer review process](https://www.pyopensci.org/software-peer-review/our-process/how-review-works.html) run by a community of dedicated volunteers. Reviews are supportive and fully transparent with the shared goal of improving the quality, usability and maintainability of the software that is driving open science.
+
+    * Our [partnership with the Journal of Open Source Software](https://www.pyopensci.org/software-peer-review/partners/joss.html) further allows scientists to record academic credit for their work through a cross-ref enabled DOI.
+
+    * Reviews are led by diverse teams of people from different backgrounds to enhance the feedback quality."
+    title: "About peer review of scientific Python software"
+joss:
+  - image_path: images/pyopensci-people-at-computer.jpg
+    alt: "A pencil sketch of a round table with people sitting around it from different backgrounds working on laptops and also writing together."
+    excerpt: "You don't have to chose between pyOpenSci and JOSS. We have a collaboration with
+JOSS where you can submit your package to us for review. If your package is accepted
+and if it is in scope for JOSS, it will then be fast tracked through JOSS'
+review process; they will only review your paper.md file. Consider this a
+win-win!
```

Comment:
```suggestion
    excerpt: "Our collaboration with JOSS means that you don't have to chose between pyOpenSci and JOSS. Submit your package to pyOS for review, and if your package is accepted and in scope for JOSS, it will be fast-tracked through JOSS' review process; they will only review your `paper.md` file. It's a win-win!"
```

Reply:
i want to commit this one but i suspect this edit needs to be applied to lines 38-42 and here it's replacing just line 42 with that entire blog of text. i'll come back to this edit !! the text is excellent!